sentinel-adapter/sentinel-web-servlet/src/main/java/com/alibaba/csp/sentinel/adapter/servlet/config/WebServletConfig.java

```diff
@@ -41,4 +43,19 @@ public static String getBlockPage() {
     public static void setBlockPage(String blockPage) {
         SentinelConfig.setConfig(BLOCK_PAGE, blockPage);
     }
+
+    /**
+     * Return status 429 in the default block page,
+     * you can use -Dcsp.sentinel.web.servlet.block.page.http.status=200 or other http status,
+     * to set http status which you want of the default block page.
+     *
+     * @return http status
+     */
+    public static int getBlockPageHttpStatus() {
+        return Integer.parseInt(SentinelConfig.getConfig(BLOCK_PAGE_HTTP_STATUS));
```

Comment:
We need to handle the parsing error or empty value here. If an error occurs or the item is absent, then we could use the default 429.

Reply:
Make sense, thanks for guiding.
When csp.sentinel.web.servlet.block.page.http.status is empty or not number, the block page http status will be automatically set to 429(Too Many Requests).
Review code again, please~